config.xml

```diff
@@ -1,106 +1,108 @@
+  <preference name="SplashScreenSpinnerColor" value="0xffffffff" />
+  <preference name="SplashMaintainAspectRatio" value="true" />
+  <preference name="SplashShowOnlyFirstTime" value="false" />
+  <allow-navigation href="*" />
+  <allow-navigation href="http://10.40.189.168:8101" />
+  <allow-navigation href="http://10.40.189.168:8100" />
+  <allow-navigation href="http://10.40.177.195:8101" />
+  <allow-navigation href="http://10.40.177.195:8100" />
+  <allow-navigation href="http://10.40.140.19:8100" />
+  <allow-navigation href="http://172.20.10.5:8100" />
+  <allow-navigation href="http://10.41.163.140:8100" />
+  <allow-navigation href="http://10.40.230.126:8100" />
+  <allow-navigation href="http://10.40.149.14:8100" />
+  <allow-navigation href="http://10.40.149.14:8101" />
+  <allow-navigation href="http://172.20.10.5:8101" />
+  <allow-navigation href="http://10.236.23.88:8100" />
+  <allow-navigation href="http://172.20.20.20:8100" />
+  <allow-navigation href="http://10.0.0.150:8100" />
+  <allow-navigation href="http://10.35.12.253:8100" />
+  <allow-navigation href="http://10.35.12.253:8101" />
+  <allow-navigation href="http://169.254.164.7:8100" />
+  <allow-navigation href="http://10.131.139.253:8101" />
+  <allow-navigation href="http://10.131.139.253:8100" />
+  <allow-navigation href="http://10.40.221.18:8101" />
+  <plugin name="cordova-android-support-gradle-release" spec="^1.4.7">
+    <variable name="ANDROID_SUPPORT_VERSION" value="27.+" />
+  </plugin>
+  <plugin name="cordova-plugin-android-permissions" spec="^1.0.0" />
+  <plugin name="cordova-plugin-camera" spec="^3.0.0" />
+  <plugin name="cordova-plugin-console" spec="^1.1.0" />
+  <plugin name="cordova-plugin-device" spec="^2.0.2" />
+  <plugin name="cordova-plugin-dialogs" spec="^2.0.1" />
+  <plugin name="cordova-plugin-media-capture" spec="^3.0.2" />
+  <plugin name="cordova-plugin-opensmile" spec="^2.0.3" />
+  <plugin name="cordova-plugin-splashscreen" spec="^4.1.0" />
+  <plugin name="cordova-plugin-statusbar" spec="^2.4.2" />
+  <plugin name="cordova-plugin-vibration" spec="^3.1.0" />
+  <plugin name="cordova-plugin-whitelist" spec="^1.3.3" />
+  <plugin name="cordova-sqlite-storage" spec="^2.6.0" />
+  <plugin name="ionic-plugin-keyboard" spec="^2.2.1" />
+  <plugin name="phonegap-plugin-barcodescanner" spec="^7.1.2">
+    <variable name="ANDROID_SUPPORT_V4_VERSION" value="27.+" />
+  </plugin>
+  <plugin name="cordova-plugin-local-notification" spec="^0.9.0-beta.3" />
+  <plugin name="cordova-plugin-app-version" spec="^0.1.9" />
+  <plugin name="cordova-plugin-fcm" spec="git+https://github.com/yatharthranjan/cordova-plugin-fcm.git" />
+  <plugin name="cordova-plugin-globalization" spec="^1.11.0" />
+  <plugin name="cordova-plugin-firebase-analytics" spec="^1.0.1">
+    <variable name="FIREBASE_CORE_VERSION" value="16.0.+" />
+  </plugin>
+  <plugin name="phonegap-plugin-mobile-accessibility" spec="^1.0.5" />
```

Comment:
why do we use this plugin?

Reply:
This is to ignore accessibility settings because some phones have the font increase accessibility setting set which breaks the UI.